src/mscorlib/shared/System/ReadOnlySpan.cs

```diff
@@ -260,6 +247,12 @@ public override int GetHashCode()
             throw new NotSupportedException(SR.NotSupported_CannotCallGetHashCodeOnSpan);
         }
 
+        /// <summary>
+        /// Returns a <see cref="String"/> with the name of the type and the number of elements
+        /// </summary>
+        /// <returns>A <see cref="String"/> with the name of the type and the number of elements</returns>
```

Comment:
nit: Is the returns xml tag necessary? It seems redundant with whatever has been stated in the summary anyway.

Reply:
It's definitely redundant. I added it because I thought that intellisense may do something useful with the returns tag, but I couldn't actually find a case where it's used without the summary. I'll remove it.